CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ We use [Break Versioning][breakver]. The version numbers follow a `<major>.<mino
 
 Malli is in [alpha](README.md#alpha).
 
+## 0.5.0 (2021-04-13)
```

Comment:
I wasn't sure what to put here so guessed this would be a minor version bump rather than just a patch.

The date comes across as quite presumptuous of me, and can absolutely be changed! :see_no_evil: